cmd/s3-public-buckets/s3publicbuckets.go

```diff
@@ -13,11 +13,21 @@ import (
 	"github.com/aws/aws-sdk-go/service/configservice"
 )
 
+type AWSConfigService interface {
+	PutEvaluations(*configservice.PutEvaluationsInput) (*configservice.PutEvaluationsOutput, error)
+}
+
 func main() {
 	lambda.Start(handleRequest)
 }
 
 func handleRequest(ctx context.Context, configEvent events.ConfigEvent) {
+	cSession := session.Must(session.NewSession())
+	svc := configservice.New(cSession)
+	handleRequestWithConfigService(ctx, configEvent, svc)
```

Comment:
You should do error handling here, because you're returning an error. handleRequest could have a return perhaps, see https://docs.aws.amazon.com/lambda/latest/dg/golang-handler.html